client/src/components/shared/ProjectOwnerBaseProfileForm.jsx

```diff
@@ -150,6 +153,52 @@ const _ProjectOwnerBaseProfileForm = ({
               fullWidth
               margin="normal"
             />
+            {
```

Comment:
Can consider moving this whole chunk into a `render*` function

Reply:
It's a dumb component so not sure how to make a render component. I'll merge first and can ask you about it 2 Feb.